### PR TITLE
day1

### PR DESCRIPTION
**Pull request by ShayRasooly**

### CODE REVIEW

File: Week1/Day1/Daily_Challenge.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    print("Your string is too long.")
+
+print(my_str[0], my_str[-1])
+
+for letter in range(1, len(my_str) + 1):
+    print(my_str[:letter])
+
+my_list = list(my_str)
+random.shuffle(my_list)
+print(''.join(my_list))
```
Comment (ShayRasooly) [inline Week1/Day1/Daily_Challenge.py:19]: Perfect!

File: Week1/Day1/ExercisesXPGold.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#Exercise 1
+print("Hello world\n" * 4 + "I love python\n" * 4)
+
+#Exercise 2
+season = int(input("Please input a month from 1 to 12: "))
+
+if season == 12 or season == 1 or season == 2:
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXPGold.py:7]: must be a more elegant way to check if a number is inside range...

File: Week1/Day1/ExercisesXPGold.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    print("This is a Winter season.")
+elif season == 3 or season == 4 or season == 5:
+    print("This is a Spring season.")
+elif season == 6 or season == 7 or season == 8:
+    print("This is a Summer season.")
+elif season == 9 or season == 10 or season == 11:
+    print("This is a Autumn season.")
+else:
+    print("The number is out of the rage. Please try again.")
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXPGold.py]: out*rage*ous!

File: Week1/Day1/ExercisesXPGold.py (hunk 2 above)
Comment (zhanna-naumenko) [reply Week1/Day1/ExercisesXPGold.py:7]: season = int(input("Please input a month from 1 to 12: "))
winter = [12, 1, 2]
spring = range(3, 6)
summer = range(6, 9)
autumn = range(9, 12)

if season in winter:
    print("This is a Winter season.")
elif season in spring:
    print("This is a Spring season.")
elif season in summer:
    print("This is a Summer season.")
elif season in autumn:
    print("This is a Autumn season.")
else:
    print("The number is out of the range. Please try again.")

File: Week1/Day1/ExercisesXPGold.py (hunk 3 above)
Comment (zhanna-naumenko) [reply Week1/Day1/ExercisesXPGold.py]: elif season == 9 or season == 10 or season == 11:
    print("This is a Autumn season.")
else:
    print("The number is out of the range. Please try again.")

**Pull request merged**